style.css

```diff
@@ -0,0 +1,100 @@
+html{
+    height: 100%;
+}
+
+body{
+    padding-top: 4%;
+    color: white;
+    background-image:linear-gradient(
+        90deg,
+        rgba(58, 58, 158, 0.8),
+        rgba(136, 136, 206, 0.7)
+    ),
+    url(/tribute-page-main-image.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-attachment: fixed;
+}
+
+h1, p {
+    margin-top: 0;
+    margin-bottom: 1rem;
+}
+
+#description{
+    font-style: italic;
+    font-weight: 200;
+}
+
+.text{
+    text-align: center;
+}
+
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+.input-area{
+    min-height: 120px;
+    width: 100%;
+    padding: 0.625rem;
+    resize: vertical;
+}
+
+form{
+    background: rgba(27, 27, 50, 0.8);
+    display: block;
+    margin: 0 auto;
+    border-radius: 10px;
+    max-width: 42%;
+}
+
+label{
+    display: flex;
+    font-size: 1.125rem;
+    margin-bottom: 0.5rem;
+    box-sizing: border-box;
+}
+
+.form-group{
+    margin: 0 auto 1.25rem auto;
+    padding: 0.25rem;
+}
+
+.form-control{
+    display: block;
+    width: 100%;
+    height: 2.375rem;
+    padding: 0.375rem 0.75rem;
+    color: #495057;
+    background-color: #fff;
+    background-clip: padding-box;
+    border: 1px solid #ced4da;
+    border-radius: 0.25rem;
+    transition: border-color 0.15s ease-in-out;
+    box-sizing: border-box;
+}
+
+.input-radio, .input-checkbox {
+    display: inline-block;
+    margin-right: 0.625rem;
+    min-height: 1.25rem;
+    min-width: 1.25rem;
+}
+
+#submit{
+    background: gray;
+    border: none;
+    border-radius: 2px;
+    margin-left: 5px;
+    height: 2rem;
+    width: 15%;
+    font-weight: bold;
+}
+
+@media (min-width: 480px){
+form{
+    padding: 2.5rem
```

Comment:
[ ] The comment should be phrased politely, as suggestions instead of orders. `For example:"I think it would be better to use semantic tags"`.kindly fix that 🤓.
[ ] always include an explanation, a "why", for each requested change. `For example: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible.` kindly fix that🤓